src/tribol/interface/mfem_tribol.cpp

```diff
@@ -127,11 +133,14 @@ void setMfemLORFactor( integer cs_id, integer lor_factor )
    coupling_scheme->getMfemMeshData()->SetLORFactor(lor_factor);
 }
 
-void setMfemKinematicConstantPenalty( integer cs_id, 
-                                      real mesh1_penalty,
-                                      real mesh2_penalty )
+void setMfemKinematicConstantPenalty( IndexT cs_id, 
+                                      RealT mesh1_penalty,
+                                      RealT mesh2_penalty )
 {
-   auto coupling_scheme = CouplingSchemeManager::getInstance().getCoupling(cs_id);
+   auto coupling_scheme = CouplingSchemeManager::getInstance().findData(cs_id);
+   SLIC_ERROR_ROOT_IF( !coupling_scheme, 
```

Comment:
Maybe a new method in `DataManager` could handle this?  We would call `findDataAndCheck(cs_id)` and it would report an error if the cs_id isn't in the map.   This might be better to defer for a more complete overhaul of error reporting?